x/dsmr/storage_test.go

```diff
@@ -121,10 +122,10 @@ func TestStoreAndSaveValidChunk(t *testing.T) {
 	chunkCerts := storage.GatherChunkCerts()
 	require.Empty(chunkCerts)
 
-	chunkCert := &ChunkCertificate{
+	chunkCert := &ChunkCertificate[tx]{
```

Comment:
Should we modify these tests to account for when the signature should be present or would you prefer that's tested through the node tests?

Reply:
We're now exporting the `ChunkStorage` type outside of this package, but i don't think there's a use-case where people should be manually playing with the internals of the database. I think I would prefer that we test this through the `Node` api